### ISSUE TITLE
Pane does not resize with figure

### ISSUE THREAD

**Issue by spurra** [facebookresearch/visdom#96]
I've got the following code:

```
vis = require 'visdom'()
vis:line{
    Y = torch.Tensor(error01):t(),
    X = torch.range(1, #error01[1]),
    options = {
        legend = {'Training', 'Validation'},
        title    = '01 Loss',
        xmin = 0,
        xmax = 1,
        width = #error01[1],
        marginleft   = 30,
        marginright  = 30,
        marginbottom = 80,
        margintop    = 30,
    }
}

```

The figure resizes successfully, but the pane it is in remains the same size. Is there a way to programmatically change it or make it adapt?

Edit: This is what I mean
http://oi66.tinypic.com/28btts5.jpg

**Comment by ajabri** [comment]
Hi @spurra!

Yes, this feature will be added very soon. It has already been implemented on a local branch, just needs to be pushed out.

**Comment by spurra** [comment]
@ajabri Thanks for your rapid response. Would it be possible to ping me once its in?

**Comment by ajabri** [comment]
Addressed in 0c494b0
